include/secp256k1_musig.h

```diff
@@ -7,364 +7,171 @@
 extern "C" {
 #endif
 
-#include <stdint.h>
+#include <stddef.h>
 
-/** This module implements a Schnorr-based multi-signature scheme called MuSig
- * (https://eprint.iacr.org/2018/068.pdf). It is compatible with bip-schnorr.
+/** This module implements a Schnorr-based multi-signature scheme called MuSig2
+ * (https://eprint.iacr.org/2020/1261, see Appendix B for the exact variant).
+ * Signatures are compatible with BIP-340 ("Schnorr").
  * There's an example C source file in the module's directory
  * (src/modules/musig/example.c) that demonstrates how it can be used.
  *
- * The documentation in this include file is for reference and may not be sufficient
- * for users to begin using the library. A full description of API usage can be found
- * in src/modules/musig/musig.md
+ * The module also supports BIP-341 ("Taproot") public key tweaking and adaptor
+ * signatures as described in
+ * https://github.com/ElementsProject/scriptless-scripts/pull/24 (see
+ * https://bitcoinops.org/en/topics/adaptor-signatures/ for a basic introduction
```

Comment:
(No action needed) Oops, I only skimmed the discussion/comments of the PR and not the changed files. The explanation in `musig2-adaptorsig.md` is really good, and I wouldn't have asked for another reference if I saw that right away.